apps/web/components/AnchorNavigation/AnchorNavigation.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useMemo, useState } from 'react'
+import useScrollSpy from '../../hooks/useScrollSpy'
+import { Bullet } from '../Bullet/Bullet'
+import { Text, Box, FocusableBox, Stack } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
+
+export interface AnchorNavigationProps {
+  title?: string
+  navigation: Array<{ id: string; text: string }>
+  position: 'left' | 'right'
+}
+
+export const AnchorNavigation = ({
+  title,
+  navigation,
+  position = 'left',
+}: AnchorNavigationProps) => {
+  const ids = useMemo(() => navigation.map((x) => x.id), [navigation])
+  const [activeId, navigate] = useScrollSpy(ids)
+  const [bulletRef, setBulletRef] = useState<HTMLElement>(null)
+  const { width } = useWindowSize()
+  const [isDesktop, setIsDesktop] = React.useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width >= theme.breakpoints.lg) {
+      return setIsDesktop(true)
+    }
+    setIsDesktop(false)
+  }, [width])
+
+  return (
+    <Box
+      position="relative"
+      marginLeft={[0, 0, 0, 4]}
+      paddingBottom={[0, 0, 0, 6]}
+      paddingLeft={[3, 3, 4]}
+      borderLeftWidth={'standard'}
+      borderColor={'blue200'}
+      borderStyle="solid"
+    >
+      {bulletRef && isDesktop && (
```

Comment:
https://github.com/island-is/island.is/pull/2125/files#r537410684 same here regarding media queries rather than js event listeners